network_observability/metrics-alerts-dashboards.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="metrics-dashboards-alerts"]
+= Using metrics, dashboards, and alerts
+include::_attributes/common-attributes.adoc[]
+:context: metrics-dashboards-alerts
+
+toc::[]
+
+The Network Observability Operator uses the `flowlogs-pipeline` to generate metrics out of flow logs. You can make use these metrics by setting custom alerts and viewing dashboards.
```

Comment:
* If it maintains the same correct meaning, one word instead of two
* "of" is missing with the "make use of". Or there's probably a better singular word that could work too. I just can't think of a good one to suggest at the moment :) Maybe "utilize", though I know a lot of times people say "utilize" you can just mean "use". But maybe it would be appropriate here :shrug: 

```suggestion
The Network Observability Operator uses the `flowlogs-pipeline` to generate metrics from flow logs. You can utilize these metrics by setting custom alerts and viewing dashboards.
```